qiskit/converters/circuit_to_dagcanonical.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+'''Helper function for converting a circuit to a dag canonical'''
+
+from qiskit.dagcircuit.dagcanonical import DAGcanonical
```

Comment:
whatever name you go with, make sure it is properly CamelCased. In this case it should be `DAGCanonical`.